app/services/policies/policies_list_service.rb

```diff
@@ -6,11 +6,16 @@
 class Policies::PoliciesListService
   def self.call(account, builtin: true)
     list = PolicyList.new
-    if builtin
-      response = ::JSONClient.get(ThreeScale.config.sandbox_proxy.apicast_registry_url)
-      return unless response.ok?
-
-      list.merge!(PolicyList.from_hash(response.body['policies']))
+    apicast_registry_url = ThreeScale.config.sandbox_proxy.apicast_registry_url
+    if builtin && apicast_registry_url
+      begin
```

Comment:
Instead of begin/rescue, better either rescue the whole method or make a method for this part, and I'd say in this case it should be inside this same method.